### DISCUSSION TITLE
Add small functionnality to clean up sql scripts ?

### DISCUSSION THREAD

**Discussion by ravenlost** [dbeaver/dbeaver#11880] [DBeaver Development]
Hello,

Just wondering if it would be possible to add a small functionality in the SQL Editor menu > Open SQL script as to be able to right-click on an SQL script and DELETE, and why not RENAME while we're at it? I haven't yet found how to clean ups all my temp scripts from there, without going into my drive's folder... 

![image](https://user-images.githubusercontent.com/62449156/112475165-cd96d580-8d70-11eb-8973-9d2fac33c3cb.png)

Thanks. Pat

**Comment by LonwoLonwo** [comment]
As a workaround, you can rename and delete scripts here 
![2021-03-26 00_56_21-DBeaver 7 3 2 - _PostgreSQL - dvdrental_ Script-33](https://user-images.githubusercontent.com/45152336/112549075-45471d80-8dce-11eb-942f-8232c89d6525.png)
